tools/start-emulators.ps1

```diff
@@ -118,7 +118,7 @@ if (!$SkipCosmosDBEmulator -and $startedCosmos -eq $true)
     $cosmosStatus = Get-CosmosDbEmulatorStatus
     Write-Host "CosmosDB emulator status: $cosmosStatus"
 
-    $waitSuccess = Wait-CosmosDbEmulator -Status Running -Timeout 60
+    $waitSuccess = Wait-CosmosDbEmulator -Status Running -Timeout 60 -ErrorAction Continue
```

Comment:
Could this lead to difficult to diagnose test failures?

Reply:
no -- it'll fail again later if the emulator doesn't start successfully or it times out (this is what happens today) -- but this was short-circuiting the retry logic and failing too early